templates/docker-init.j2

```diff
@@ -1,12 +1,61 @@
 # Generated by Ansible for {{ansible_fqdn}}
 description "Docker daemon"
 
-start on filesystem
+start on (filesystem and net-device-up IFACE!=lo)
 stop on runlevel [!2345]
+limit nofile 524288 1048576
```

Comment:
Seems too arbitrary. Would be nice to control what limits to use or at least these limit values.